microsite/data/plugins/argo-cd.yaml

```
@@ -0,0 +1,9 @@
+---
+title: Argo CD
+author: roadie.io
+authorUrl: https://roadie.io
+category: CI
+description: View ArgoCD status for your projects in Backstage.
```

```suggestion
description: View Argo CD status for your projects in Backstage.
```

Nit to align with the title 😄  And also corrct per title believe the project calls itself "Argo CD" with a space:
https://argoproj.github.io/argo-cd/

---

Done